ci/do_ci.sh

```diff
@@ -6,6 +6,26 @@ set -e
 
 echo "building using $NUM_CPUS CPUs"
 
+if [[ "$1" == "bazel.debug" ]]; then
+  echo "debug bazel build with tests..."
+  cd ci
+  ln -sf /thirdparty prebuilt
+  ln -sf /thirdparty_build prebuilt
+  # Not sandboxing, since non-privileged Docker can't do nested namespaces.
+  echo "Building..."
+  export CC=gcc-4.9
+  export CXX=g++-4.9
+  TEST_TMPDIR=/source/build USER=bazel bazel build --strategy=CppCompile=standalone \
```

Comment:
I've switched to --batch by default and cleaned up the duplication (bazelrc is probably a bit overkill so just refactored).